### PR TITLE
web page in ressemblence to youtube video player 

### PR DESCRIPTION
**Pull request by ABDELLANI-Youcef**
# HTML embedding video and images
In this project we:
- add a header to the youtube video player page.
- add main section that contains:
	- youtube video player
	- video description
- add a sidebar that contains images for proposed videos with their titles, their publisher and views numbers
- add a style sheet to the page

[reference to old repository](https://github.com/ABDELLANI-Youcef/EmbeddingVideo)

### CODE REVIEW

File: assets/style.css [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+/*#youtube{
+    
+}*/
+.user{
+    width: 35px;
+    height: 35px;
+    border-radius: 50%;
+    
+}
+#subscribe{
```
Comment (expjazz) [inline assets/style.css]: You repeated this selector again on line 108. Please group them together on the next projects.

**Review by expjazz — CHANGES_REQUESTED** [review]
## Hey team

### [changes required]

#### Your project looks really great, but we still need to change a thing things to get it approved.

- [ ] The logo in the navbar should have a little more width so it looks more like the original. The search bar could be a little bigger.

| Yours | Original |
|:---:|:---:|
| image | image

- [ ] The height of the embedded video is a little too much. 

- [ ] As you can In the marks see below, you duplicated one css selector. Kindly fix this.

- [optional ] You used too many ids. For a small project like this, it is fine, but it can get hard to handle especially if we have elements that repeat in other places on the page. Since id's need to be unique, it is better to use classes instead.

#### That is it. As I said, the project looks really good. Good job team! :clap: 

### Please submit for a new code review once you are ready

Good Luck and Happy Coding :pray:

P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:

**Comment by ABDELLANI-Youcef** [comment]
Thank you for your review. We made the changes that you told us to do.

**Review by expjazz — APPROVED** [review]
## Great job on the changes requested! The project is approved :clap: 

Good Luck on the next projects and Happy Coding :pray:

P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:

**Pull request merged**